digits/model/tasks/caffe_train.py

```diff
@@ -807,6 +813,26 @@ def task_arguments(self, resources, env):
         if self.pretrained_model:
             args.append('--weights=%s' % ','.join(map(lambda x: self.path(x), self.pretrained_model.split(':'))))
 
+        if platform.system() == 'Windows' and including_python_layer:
+            if len(identifiers) > 1:
+                    self.logger.warning('Python layer can not work in multiple GPU.  Will use the first GPU only')
```

Comment:
I think you're getting confused about which issue you're trying to solve.

1. multi-GPU + Python layer fails on all systems. We don't need to check for this - let the framework deal with it.
2. single-GPU + Python layer fails on Windows unless you run it with pycaffe.

Don't try to solve (1) in this pull request.